README.md

```diff
@@ -98,6 +98,14 @@ Optional parameters are also... an option:
     ],
 ```
 
+By default it only redirects if the request has a ```404``` response code but it's possible to be redirected on any response code. 
+To achieve this you may change the ```status_code``` option to an array of response codes or leave it empty if you wish to be redirected no matter what the response code was sent to the URL.
+You may override this using the following syntax to achieve this:  
+
+```php
+    'status_code' => [\Symfony\Component\HttpFoundation\Response::HTTP_NOT_FOUND],
```

Comment:
Let's name this option to `redirect_status_codes`. That makes it a bit more clear.